homeassistant/components/rdw/sensor.py

```diff
@@ -43,13 +43,13 @@ class RDWSensorEntityDescription(
         key="apk_expiration",
         name="APK Expiration",
         device_class=DEVICE_CLASS_DATE,
-        value_fn=lambda vehicle: vehicle.apk_expiration.isoformat(),
+        value_fn=lambda vehicle: vehicle.apk_expiration,
```

Comment:
Do we know if this is a date or datetime ?

Reply:
Yes, this integration is fully typed (strictly typed), including the package (I wrote the package).